### ISSUE TITLE
Bluetooth

### ISSUE THREAD

**Issue by Jdurbisgay** [Duwaynef/MLM2PRO-BT-APP#5]
I can't get my mlm2pro to connect to my pc

**Comment by Jdurbisgay** [comment]
Do you need windows 11?

**Comment by Jdurbisgay** [comment]
And can I use my Ethernet ?

**Comment by Jdurbisgay** [comment]
Yes it won’t pop up

**Comment by Jdurbisgay** [comment]
on the bluetooth on my pc it just will not pop up

**Comment by Duwaynef** [comment]
are you on windows 11? if so make sure advanced bluetooth discovery is enabled.

**Comment by Jdurbisgay** [comment]
i am on windows 10

**Comment by Jdurbisgay** [comment]
windows 10

**Comment by Duwaynef** [comment]
This is the first time i'm hearing of it not appearing under bluetooth devices.

are you under bluetooth and hitting add bluetooth device right? / your pc has a bluetooth adapter built in?

**Comment by Jdurbisgay** [comment]
yes it had adapter and yes im on bluetooth i have everything set up it just wont show up when im trying to connect it

**Comment by Duwaynef** [comment]
very strange. its the first time i'm hearing of it not showing up in bluetooth devices.  does other bluetooth devices show up when you are searching?

and do you happen to know what version of bluetooth your adapter supports?

**Comment by Jdurbisgay** [comment]
I don’t know version and yes other stuff pops up

**Comment by Jdurbisgay** [comment]
Hello

**Comment by Duwaynef** [comment]
Give the springbok connector a try, it don't require the device to be paired with bluetooth.

**Comment by Jdurbisgay** [comment]
I bought an r10 and it won’t connect either

**Comment by cappy175** [comment]
@Duwaynef  I am unable to get my pc to connect to the MLM2-MP1... It will however connect to the BT one but I know that is not the correct one. I did remove it from the list. It will still not connect to my PC. What now? BTW I did the activation steps with the rapsodo app and awesome golf.
